### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by kveselits**
I messed up and had to undo the first rebase I did. I think that's why there are so many commits. Wasn't sure how to remedy this.

### CODE REVIEW

**Comment by 7hawks** [comment]
I will review this.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# ASP.NET Core (.NET Framework)
+# Build and test ASP.NET Core projects targeting the full .NET Framework.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/dotnet-core
+
+trigger:
+- master
```
Comment (Keboo) [inline azure-pipelines.yml]: You probably want this to trigger on your assignment branches.
```suggestion
- Assignment*
```

**Comment by kveselits** [comment]
@Keboo -- forgot to include [the link to my pipeline for this assignment](https://dev.azure.com/kveselits/EWU-CSCD379-2020-Winter/_build?definitionId=4&_a=summary)

**Comment by Keboo** [comment]
@kveselits can you send me the output from your VSTest@2 task? ADO has a bug that makes it difficult to view.

**Comment by kveselits** [comment]
Is this the log you’re talking about? https://dev.azure.com/kveselits/dc23bf8a-fb4d-4048-955b-5dcc105d8294/_apis/build/builds/21/logs/9

**Comment by Keboo** [comment]
@kveselits yes thank you. I have updated my comment, the unit tests are configured but the tests are not actually getting run.

**Comment by kveselits** [comment]
@Keboo -- I understand if it's a bit too late for these changes, but I believe I have everything working now. In trying to fix the problem I ended up deleting the pipeline and creating a new one.

[Link to my [new] pipeline](https://dev.azure.com/kveselits/EWU-CSCD379-2020-Winter/_build?definitionId=8&_a=summary)

[Link to log file for VSTest](https://dev.azure.com/kveselits/dc23bf8a-fb4d-4048-955b-5dcc105d8294/_apis/build/builds/44/logs/8)

**Comment by Keboo** [comment]
@kveselits yea a bit too late for Assignment 1, but yes that is exactly what is expected.

**Pull request closed without merging**